tests/boot/boot_from_pxe.pm

```diff
@@ -50,6 +50,7 @@ sub run {
         $openqa_url = 'http://' . $openqa_url unless $openqa_url =~ /http:\/\//;
         my $repo = $openqa_url . "/assets/repo/${image_name}";
         $image_path = "$path/linux initrd=$path/initrd install=$repo";
+        $image_path .= '?device=eth0' if (check_var('BACKEND', 'ipmi'));
```

Comment:
I think we should introduce another variable for the network interface (`eth0`) here. Maybe with the default-fallback to `eth0`.